tests/test_compose.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+
+import numpy as np
+import torch
+from numpy.testing import assert_almost_equal, assert_array_equal
+
+from torch_audiomentations import PolarityInversion, Compose, PeakNormalization
+from torch_audiomentations.augmentations.gain import Gain
+from torch_audiomentations.utils.dsp import convert_decibels_to_amplitude_ratio
+
+
+class TestCompose(unittest.TestCase):
+    def test_compose(self):
+        samples = np.array([[[1.0, 0.5, -0.25, -0.125, 0.0]]], dtype=np.float32)
+        sample_rate = 16000
+
+        augment = Compose(
+            [
```

Comment:
Nitpicking...
```suggestion
            transforms=[
```

Reply:
Some people will use an arg, some people will use a kwarg :) I added `transforms=` in some of the tests now, so we cover both